airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -346,6 +358,16 @@ def _build_cluster_data(self):
             if not self.single_node:
                 cluster_data['worker_config']['image_uri'] = custom_image_url
 
+        elif self.custom_image_family:
+            project_id = self.custom_image_project_id or self.project_id
+            custom_image_url = (
+                'https://www.googleapis.com/compute/beta/projects/'
+                '{}/global/images/family/{}'.format(project_id, self.custom_image_family)
```

Comment:
Let's use f-string here
```suggestion
                f'{project_id}/global/images/family/{self.custom_image_family}'
```